### PR TITLE
Forgot password and otp screen

### PR DESCRIPTION
**Pull request by jayminrao**

### CODE REVIEW

File: lib/Presentation/Authentication/registration_screen.dart [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            TextformfieldWidget(
+              controller: emailNameController,
+              hint: 'darshan12@gmail.com',
+            ),
+            SizedBox(height: context.height(context) * 0.020),
+            const Text(AppStrings.registrationScreenUserName,
+                style: TextStyle(fontWeight: FontWeight.w600, fontSize: 16)),
+            TextformfieldWidget(
+              controller: userNameController,
+              hint: '********',
```
Comment (RAWAN3830) [inline lib/Presentation/Authentication/registration_screen.dart:56]: use string file , don't use static string

File: lib/Presentation/Authentication/login_screen.dart [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            TextformfieldWidget(
+              controller: emailNameController,
+              hint: 'darshan12@gmail.com',
+            ),
+            SizedBox(height: context.height(context) * 0.020),
+            const Text(AppStrings.loginScreenPassword,
+                style: TextStyle(fontWeight: FontWeight.w600, fontSize: 16)),
+            TextformfieldWidget(
+              controller: passWordController,
+              hint: '********',
```
Comment (nikkielevancex) [inline lib/Presentation/Authentication/login_screen.dart:57]: add some meaningful hint text

File: lib/Presentation/Authentication/login_screen.dart [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+              ],
+            ),
+            SizedBox(height: context.height(context) * 0.025),
+            const CommonListViewBuliderWidget(),
+            SizedBox(height: context.height(context) * 0.025),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                const Text(AppStrings.dontHaveAnAccount,
+                    style: TextStyle(fontSize: 16)),
```
Comment (nikkielevancex) [inline lib/Presentation/Authentication/login_screen.dart:106]: suggestion: use textStyle from Theme

File: lib/Presentation/Forgotpassword/Reset_Password_Screen.dart [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+            SizedBox(height: context.height(context) * 0.37),
+            GestureDetector(
+              onTap: () {
+                showModalBottomSheet(
+                  context: context,
+                  builder: (context) {
+                    return Column(
+                      children: [
+                        Container(
+                          height: 500,
```
Comment (nikkielevancex) [inline lib/Presentation/Forgotpassword/Reset_Password_Screen.dart:74]: avoid using static height

**Pull request closed without merging**